src/pages/CreateAssignment/CreateAssignment.js

```diff
@@ -49,9 +49,46 @@ function CreateAssignment(props) {
     setLessonCreationDate(originalDate);
   }, [firebase]);
 
+  async function parseCardsFromLesson(lesson) {
+    let lessonCards = {};
+    let deploymentAccountIds = new Set();
+    for (const dueDate in lesson.dueDates) {
+      for (const deploymentAccount of lesson.dueDates[dueDate]) {
+        deploymentAccountIds.add(deploymentAccount);
+      }
+    }
```

Comment:
it would be more readable to use `.forEach()` instead of regular `for` loops

Reply:
ok i can fix this

Reply:
on second thought, changing it to forEach() doesn't do much for readability, it just changes it to this:
`lesson.dueDates.forEach(date => {
    lesson.dueDates.date.forEach(deploymentAccount => {
         deploymentAccountIds.add(deploymentAccount);
    }
});`

imo the above can be more confusing / not much of an improvement ^^
lmk if you still think that's better though and I can change it

Reply:
i'm just more inclined to `forEach` so i'd prefer that, i feel like `for` loops in javascript are kind of rare